### ISSUE TITLE
Handling two plural words in a sentence

### ISSUE THREAD

**Issue by pswai** [rubenv/angular-gettext#172]
Is there any way to translate a single sentence with two or more plural words? For example, `You obtained 2 apples and 3 oranges!`

Right now, what I can think of is to break the sentence into two simple sentences. However it would be really helpful if we can make the conjunction work. :)

**Comment by rubenv** [comment]
Nope, you can't. This is possible with more advanced translation formats like MessageFormat, but those are just incredibly painful to use for users.
